index.js

```diff
@@ -1,93 +1,53 @@
-const fs = require("fs"),
-    http = require("http"),
-    path = require("path"),
-    methods = require("methods"),
-    express = require("express"),
-    bodyParser = require("body-parser"),
-    session = require("express-session"),
-    cors = require("cors"),
-    passport = require("passport"),
-    errorhandler = require("errorhandler"),
-    mongoose = require("mongoose");
+import express from 'express';
+import bodyParser from 'body-parser';
+import swaggerJSDoc from 'swagger-jsdoc';
+import swaggerui from 'swagger-ui-express';
+//import swaggerDoc from './docs/swagger';
 
-const isProduction = process.env.NODE_ENV === "production";
 
 // Create global app object
 const app = express();
 
-app.use(cors());
+// swagger definition
+const swaggerDefinition = {
+    info: {
+      title: 'Authors Haven',
+      version: '1.0.0',
+      description: 'A Social platform for the creative at heart',
+    },
+    host: '',
+    basePath: '/',
+  };
 
-// Normal express config defaults
-app.use(require("morgan")("dev"));
-app.use(bodyParser.urlencoded({ extended: false }));
-app.use(bodyParser.json());
-
-app.use(require("method-override")());
-app.use(express.static(__dirname + "/public"));
-
-app.use(
-    session({
-        secret: "authorshaven",
-        cookie: { maxAge: 60000 },
-        resave: false,
-        saveUninitialized: false
-    })
-);
-
-if (!isProduction) {
-    app.use(errorhandler());
-}
+// options for swagger jsdoc 
+const options = {
+    swaggerDefinition: swaggerDefinition,
+    apis: ['./routes/*.js']
```

Comment:
update the pattern to recursive search any js file within the directory:
```suggestion
    apis: ['./routes/**/*.js']
```